<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>This file is part of the xTuple ERP: PostBooks Edition, a free and
open source Enterprise Resource Planning software suite,
Copyright (c) 1999-2014 by OpenMFG LLC, d/b/a xTuple.
It is licensed to you under the Common Public Attribution License
version 1.0, the full text of which (including xTuple-specific Exhibits)
is available at www.xtuple.com/CPAL.  By using this software, you agree
to be bound by its terms.</comment>
 <class>miscCheck</class>
 <widget class="QWidget" name="miscCheck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Miscellaneous Payment</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>498</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="_save">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_close">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="_recipGroup">
     <property name="title">
      <string>Write Payment To:</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QRadioButton" name="_vendRB">
          <property name="text">
           <string>Vendor</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="_custRB">
          <property name="text">
           <string>Customer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="_taxauthRB">
          <property name="text">
           <string>Tax Authority</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QStackedWidget" name="_widgetStack">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>75</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="_vendorPage">
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="margin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="VendorCluster" name="_vend">
              <property name="focusPolicy">
               <enum>Qt::StrongFocus</enum>
              </property>
              <property name="label">
               <string>Vendor #:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="_custPage">
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="margin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="CustCluster" name="_cust">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="focusPolicy">
               <enum>Qt::StrongFocus</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="_taxauthPage">
         <layout class="QGridLayout">
          <property name="margin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="_taxauthLit">
            <property name="text">
             <string>Tax Authority #:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>_taxauth</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="XComboBox" name="_taxauth">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="type">
             <enum>XComboBox::TaxAuths</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <spacer>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2">
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="_chargeToGroup">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Charge To:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_1">
      <item row="0" column="0" colspan="2">
       <widget class="QRadioButton" name="_expense">
        <property name="text">
         <string>Expense Category</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="2">
       <widget class="ExpenseCluster" name="_expcat">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="label">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QRadioButton" name="_applytocm">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Apply to Credit Memo</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" rowspan="2">
       <widget class="AropenCluster" name="_cmCluster">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="allowedDocTypes">
         <enum>AropenLineEdit::CreditMemo</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QRadioButton" name="_memo">
        <property name="text">
         <string>Create Credit Memo</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <spacer>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QGridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="_dateLit">
       <property name="text">
        <string>Date:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="DLineEdit" name="_date"/>
     </item>
     <item row="0" column="2" colspan="2">
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>261</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="_bankaccntLit">
       <property name="text">
        <string>Bank Account:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="XComboBox" name="_bankaccnt">
       <property name="type">
        <enum>XComboBox::APBankAccounts</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="_amountLit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>Amount:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="CurrCluster" name="_amount">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="currencyEditable" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>191</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="_forLit">
       <property name="text">
        <string>Memo:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="XLineEdit" name="_for"/>
     </item>
     <item row="2" column="3">
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QFrame" name="_frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="_notesLit">
          <property name="text">
           <string>Notes:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="XTextEdit" name="_notes">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="spellEnable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="5" margin="5"/>
 <customwidgets>
  <customwidget>
   <class>AropenCluster</class>
   <extends>QWidget</extends>
   <header>aropencluster.h</header>
  </customwidget>
  <customwidget>
   <class>CurrCluster</class>
   <extends>CurrDisplay</extends>
   <header>currcluster.h</header>
  </customwidget>
  <customwidget>
   <class>CurrDisplay</class>
   <extends>QWidget</extends>
   <header>currcluster.h</header>
  </customwidget>
  <customwidget>
   <class>CustCluster</class>
   <extends>QWidget</extends>
   <header>custcluster.h</header>
  </customwidget>
  <customwidget>
   <class>DLineEdit</class>
   <extends>QWidget</extends>
   <header>datecluster.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ExpenseCluster</class>
   <extends>QWidget</extends>
   <header>expensecluster.h</header>
  </customwidget>
  <customwidget>
   <class>VendorCluster</class>
   <extends>QWidget</extends>
   <header>vendorcluster.h</header>
  </customwidget>
  <customwidget>
   <class>XComboBox</class>
   <extends>QComboBox</extends>
   <header>xcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>XLineEdit</class>
   <extends>QLineEdit</extends>
   <header>xlineedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>XTextEdit</class>
   <extends>QTextEdit</extends>
   <header>xtextedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_vendRB</tabstop>
  <tabstop>_custRB</tabstop>
  <tabstop>_taxauthRB</tabstop>
  <tabstop>_vend</tabstop>
  <tabstop>_expense</tabstop>
  <tabstop>_expcat</tabstop>
  <tabstop>_applytocm</tabstop>
  <tabstop>_memo</tabstop>
  <tabstop>_date</tabstop>
  <tabstop>_amount</tabstop>
  <tabstop>_for</tabstop>
  <tabstop>_bankaccnt</tabstop>
  <tabstop>_notes</tabstop>
  <tabstop>_close</tabstop>
  <tabstop>_cust</tabstop>
  <tabstop>_taxauth</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>_close</sender>
   <signal>clicked()</signal>
   <receiver>miscCheck</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>820</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_date</sender>
   <signal>newDate(QDate)</signal>
   <receiver>_amount</receiver>
   <slot>setEffective(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_memo</sender>
   <signal>toggled(bool)</signal>
   <receiver>_expcat</receiver>
   <slot>setReadOnly(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>592</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>804</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_expense</sender>
   <signal>toggled(bool)</signal>
   <receiver>_expcat</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>581</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>804</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_custRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>_applytocm</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>606</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_taxauthRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>_memo</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>592</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_vendRB</sender>
   <signal>clicked()</signal>
   <receiver>_expense</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_taxauthRB</sender>
   <signal>clicked()</signal>
   <receiver>_expense</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_applytocm</sender>
   <signal>toggled(bool)</signal>
   <receiver>_cmCluster</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>548</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>661</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
